conan/tools/cmake/toolchain/blocks.py

```diff
@@ -547,6 +547,54 @@ def context(self):
         return {"paths": [ut.replace("\\", "/") for ut in user_toolchain]}
 
 
+class ExtraFlagsBlock(Block):
+    """This block is adding flags directly from user [conf] section"""
+
+    template = textwrap.dedent("""
+        {% if cxxflags %}
+        string(APPEND CONAN_CXX_FLAGS
+        {% for cxxflag in cxxflags %}
+        " {{ cxxflag }}"
+        {% endfor %}
+        )
```

Comment:
Minor detail, but maybe we could improve formatting of this template, it formats like:

```cmake
string(APPEND CONAN_CXX_FLAGS
" -Wpedantic"
" -fstack-protector-strong"
)
```
that is very different from other variables set in the toolchail

```suggestion
        {% if cxxflags %}
        string(APPEND CONAN_CXX_FLAGS "{% for cxxflag in cxxflags %} {{ cxxflag }}{% endfor %}")
        {% endif %}
```